src/EmbedConnection.ts

```diff
@@ -117,50 +121,86 @@ export class EmbedConnection implements ILookerConnection {
     id,
     pushHistory,
     options,
+    params = {},
   }: LoadIdParams): Promise<void> {
+    const urlParams = stringify(params)
+    const sep = urlParams ? '?' : ''
     return this.loadUrl({
       options,
       pushHistory,
-      url: `/embed/${type}/${id}`,
+      url: `/embed/${type}/${id}${sep}${urlParams}`,
     })
   }
 
   async loadDashboard(
-    id: string,
+    idOrParams: string | LoadParams,
     pushHistory?: boolean,
     options?: IConnectOptions
   ) {
     if (this._embedClient.isPageLoadEventSupported) {
-      return this.loadId({
-        id,
-        options,
-        pushHistory,
-        type: 'dashboards',
-      })
+      const loadIdParams: LoadIdParams =
+        typeof idOrParams === 'string'
+          ? {
+              id: idOrParams,
+              options,
+              pushHistory,
+              type: 'dashboards',
+            }
+          : { ...idOrParams, type: 'dashboards' }
+      return this.loadId(loadIdParams)
     }
     switch (this._pageType) {
       case 'dashboards':
         return this.sendAndReceive('dashboard:load', {
-          id,
+          id: idOrParams,
           pushHistory: pushHistory || false,
         })
     }
   }
 
   async loadExplore(
-    id: string,
+    idOrParams: string | LoadParams,
     pushHistory?: boolean,
     options?: IConnectOptions
   ) {
-    id = id.replace('::', '/') // Handle old format explore ids.
-    return this.loadId({ id, options, pushHistory, type: 'explore' })
+    const loadIdParams: LoadIdParams =
+      typeof idOrParams === 'string'
+        ? {
+            id: idOrParams,
+            options,
+            pushHistory,
+            type: 'explore',
+          }
+        : { ...idOrParams, type: 'explore' }
+    loadIdParams.id = loadIdParams.id.replace('::', '/') // Handle old format explore ids.
```

Comment:
How old is this format? I never recall this for explores even being mentioned

Reply:
I think it was there about the time the extension framework was created. It's possible I managed to conflate it with the extension framework but the code has been there since at least 2020. I'll look into it and perhaps remove. For another day though,